CMakeLists.txt

```diff
@@ -235,7 +238,7 @@ install(EXPORT miniocpp-config
         DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/miniocpp")
 
 install(FILES
-        ${MINIO_CPP_HEADERS} DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/miniocpp")
+        ${MINIO_CPP_HEADERS} DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/minio-cpp")
```

Comment:
If the library name is `miniocpp`, can't we keep it include directory as `miniocpp`?

Reply:
We can. The thing is, shouldn't the package then be called `miniocpp` instead of `minio-cpp`? Because we call it `minio-cpp`, and then we create a `miniocpp` library and `miniocpp::miniocpp` cmake targets.

I think the hyphen is doing more bad than good in this case, but I have no problem using `miniocpp` as a include subdirectory.

Reply:
BTW my goal here was to move the include files up to `/minio-cpp` and then to provide a new header `minio-cpp.h`, which would include all files in `minio-cpp` directory.

This way it would be much easier to refactor the library later without breaking user's code. Because for example the examples are wrong at the moment as they use `#include <client.h>` - which suggests that `miniocpp` is not required as a subdirectory, but that's not true and the examples only compiled because they were part of the `minio-cpp` cmake project.